examples/record-type-inclusion/record_type_inclusion.bal

```diff
@@ -25,7 +25,7 @@ public function main() {
     Employee john = {name: "John Doe", designation: "Software Engineer"};
     Employee jane = {name: "Jane Doe", designation: "UX Engineer"};
 
-    // Type referencing copies the fields including their properties
+    // Type inclusion copies the fields including their properties
     // (e.g., type, default value, optional status). As it can be seen
     // by printing an `Employee` record, the optional field company
```

Comment:
```suggestion
    // by printing an `Employee` record, the optional `company` field
```

Reply:
an "optional field" is a type of field. So not sure whether "optional company field"

Reply:
It is fine as `company` is highlighted within the back ticks.

Reply:
this was changed to "the optional field (`company`)"

Reply:
That's the suggestion.